src/components/BottomBar/BottomBar.js

```diff
@@ -42,9 +53,62 @@ export const BottomBar = () => {
 
   const selectedNote = useSelector(state => state.notesMindMap.selectedNote);
 
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+
+  const handleClick = event => {
```

Comment:
Handle which click? I think it should be something like `handleOpenMenuButtonClick`

Reply:
Also, I think usually when the menu button is clicked while the menu is open, the menu would close.
So maybe it should be `handleToggleMenuButtonClick`, and it should be toggling the menu like
```
handleToggleMenuButtonClick = event => {
  if (!isMenuOpen) {
    // if menu is not open, open it
    setAnchorEl(event.currentTarget);
  } else {
     // if menu is already open, and user clicks the button again, close the menu
     setAnchorEl(null);
}
```

Reply:
> So maybe it should be handleToggleMenuButtonClick, and it should be toggling the menu like

Seems like we actually don't need it.  In fact, the second click on the icon button will close the menu. Menu closing is achieved in another way - mui menu renders overlay and if a user clicks anywhere outside menu - menu would be closed without changing openMenu/closeMenu to toggleMenu